.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "config/plugins/unipept"]
+	path = config/plugins/unipept
+	url = https://github.com/caleb-easterly/galaxy-plugin-unipept-visualization.git
```

Comment:
This was my first shot at working with submodules...and I mostly just got confused. So I deleted .gitmodules and the submodule file in .git. Would that clean it up enough?